en/lesson-08.md

```diff
@@ -158,181 +154,217 @@ by using
   frog   & flies & small  \\
+  \bottomrule
 \end{tabular}
 \end{document}
 ```
 <!-- {% endraw %} -->
 
-Using the `\toprule`, `\midrule` and `\bottomrule` commands from the `booktabs` package we can add some
-horizontal lines.
+You may have guessed that `r` and `l` mean the rule is shortened on its *r*ight
+and *l*eft end, respectively.
+
+Sometimes a rule would be too much of a separation for two rows but to get
+across the meaning more clearly you want to separate them by some means. In this
+case you can use `\addlinespace` to insert a small skip.
 
 <!-- {% raw %} -->
 ```latex
 \documentclass{article}
 \usepackage{array}
-\usepackage{booktabs}
 
 \begin{document}
-\begin{tabular}{@{} lll @{}} \toprule
-  Animal & Food  & Size   \\ \midrule
-  dog    & meat  & medium \\
-  horse  & hay   & large  \\
-  frog   & flies & small  \\ \bottomrule
+\begin{tabular}{cp{9cm}}
+  \toprule
+  Animal & Description \\
+  \midrule
+  dog    & The dog is a member of the genus Canis, which forms part of the
+           wolf-like canids, and is the most widely abundant terrestrial
+           carnivore. \\
+  \addlinespace
+  cat    & The cat is a domestic species of small carnivorous mammal. It is the
+           only domesticated species in the family Felidae and is often referred
+           to as the domestic cat to distinguish it from the wild members of the
+           family. \\
+  \bottomrule
 \end{tabular}
 \end{document}
 ```
 <!-- {% endraw %} -->
 
-By default `\toprule` and `\bottomrule` are slightly thicker than the
-`\midrule`. The thickness of the lines can be adjusted via the optional
-parameter of these commands:
+
+## Merging cells
+
+In LaTeX you can merge cells horizontally by using the `\multicolumn` macro. It
+has to be used as the first thing in a cell. `\multicolumn` takes three
+arguments:
+
+1. The number of cells which should be merged
+2. The alignment of the merged cell
+3. The contents of the merged cell
+
+The alignment can contain anything legal in a `tabular`'s preamble, but _only a
+single column type_.
 
 <!-- {% raw %} -->
 ```latex
 \documentclass{article}
-\usepackage{array}
-\usepackage{booktabs}
+\usepackage{array, booktabs}
 
 \begin{document}
-\begin{tabular}{@{} lll @{}} \toprule[2pt]
-  Animal & Food  & Size   \\ \midrule[1pt]
+\begin{tabular}{lll}
+  \toprule
+  Animal & Food  & Size   \\
+  \midrule
   dog    & meat  & medium \\
   horse  & hay   & large  \\
-  frog   & flies & small  \\ \bottomrule[2pt]
+  frog   & flies & small  \\
+  fuath  & \multicolumn{2}{c}{unknown} \\
+  \bottomrule
 \end{tabular}
 \end{document}
 ```
 <!-- {% endraw %} -->
 
-Even more, one can set a left and right indent for the `\midrule`, so
-that this line is a little bit shorter than the `\toprule` and
-`\bottomrule` of the table.
+You can also use `\multicolumn` on a single cell to prevent whatever you defined
+in the table preamble for the current column from applying. The following uses
+this to centre the table's head row:
 
```

Comment:
I believe we are supposed to use "center" (the intro explicitly says the lessons use US spelling)

Reply:
(which I never read and I'm writing standard English most of the time, not English simplified...)